CONTRIBUTING.md

```diff
@@ -1,20 +1,191 @@
+        - [**What if you accidentally cloned using the repository URL from the HackForLA Github (instead of the fork on your Github)?**](#what-if-you-accidentally-cloned-using-the-repository-url-from-the-hackforla-github-instead-of-the-fork-on-your-github)
+            - [**i. Reset `origin` remote url**](#i-reset-origin-remote-url)
+            - [**ii. Add an `upstream` remote**](#ii-add-an-upstream-remote)
 
-- Explain your guidelines here.
+## **Setting up the local development environment**
 
+### **Creating a personal AWS account**
 
-### Working with pull requests and reviews
+- Go to [amazon.com](https://aws.amazon.com/) and click `"Sign In to the Console" > "Create a new AWS account."` 
+- Enter your email, create a password, and input your basic details. Then provide your contact information and complete the identity verification process.
+- Enter your credit/debit card information for billing purposes, opt for the free basic support.
+- Agree to the AWS Customer Agreement and Service Terms, complete the registration by clicking `"Create Account and Continue"`, verify your phone number via text or call, confirm your email address following the instructions in the confirmation email, and finally sign in to access your new AWS account using your email and password.
 
-- Explain your process.
\ No newline at end of file
+<sub>[Back to Table of Contents](#table-of-contents)</sub>
+***
+
+## **Installing Terraform**
+
+We are using the [HashiCorp Tutorials](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) for installing terraform.
+
+### **MacOS installation**
+
+In the guide we are using [homebrew](https://brew.sh) for the installation, make sure you have homebrew installed on your machine.
+
+- We will first install the `hashicorp tap` a repository of all the terraform homebrew packages, use the below command in your terminal.
+    
+    ```bash
+    brew tap hashicorp/tap
+    ```
+- Then install terraform with brew
+
+    ```bash
+    brew install terraform
+    ```
+    now confirm your installation with `terraform -help`.
+
+### **Windows installation**
+
+For windows installation we will use [choclatey](https://chocolatey.org) a free and open-source package management system for Windows.
+
+- Open the `PowerShell` terminal as an administrator and install the terraform package with 
+
+    ```bash
+    choco install terraform
+    ```
+- To confirm its installation close the current terminal and open a new `PowerShell` terminal as a normal user and use `terraform -help`.
+
+### **Linux installation**
+
+To install Terraform on your Linux distribution, visit the [HashiCorp Tutorial](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) and follow the provided instructions for your specific distribution. After installation, verify Terraform by running `terraform -help` in your terminal.
+
+<sub>[Back to Table of Contents](#table-of-contents)</sub>
+***
```

Comment:
```suggestion
[Official Hashicorp install instructions](https://developer.hashicorp.com/terraform/install)
```
These looked a little more simplified (sorry, I think I provided the wrong link before)...and then we don't have to list out each installation